tests/cmd/tuto/mod.rs

```diff
@@ -132,21 +132,19 @@ fn fixture() {
             )
             .unwrap();
 
-            let path = std::env::current_dir()
-                .unwrap()
-                .join(crate::parent_directory!());
-            let git_repo = gix::discover(path).expect("unable to find git root");
-            let git_root = git_repo
-                .path()
-                .parent()
-                .expect("failed to move from path to .git to its parent")
-                .to_str()
-                .expect("failed to convert path git root to string");
-            dbg!(git_root);
+            let project_root = std::env::current_dir().expect("Failed to get working directory");
```

Comment:
this is likely going to break

e.g. the gix code was added for a reason by @qsantos iirc

Reply:
but: if the tests pass, im happy to merge

Reply:
There was no detection for the actual root of the project. Maybe this was added since, so manually navigating to the parent directory is not needed anymore?

Reply:
This is using current working directory. So that's the directory from which it's executed.
Invoking this test using `cargo test` or `cargo nextest run` will execute it from the root of the project even if you are in a subdirectory.

If there is a reason this was here I am happy to keep it. I was trying to remove it because it wasn't working under nix. Alternatively we could just depend on git inside of the nix test